### ISSUE TITLE
How to go user section after save user profile page.

### ISSUE THREAD

**Issue by ajayrajecb** [EasyCloudConsultants/EasyCloudBooks#200]
There is not direct button shown to go user section from user profile, while in live ecb all left menu available for click and go to user section.

![2019-03-23_11-06-10](https://user-images.githubusercontent.com/46882964/54862122-d6a4d480-4d5b-11e9-93f6-be1bda33aa69.png)

**Comment by mitulpanch** [comment]
its there
![Annotation 2019-07-19 134346](https://user-images.githubusercontent.com/5259926/61520198-49347100-aa2b-11e9-85eb-ef71abc115cb.jpg)

**Comment by risipatel** [comment]
Yes, we know its there but it is hidden. So, it might cause trouble to user. @mitulpanch @ajayrajecb

**Comment by ajayrajecb** [comment]
> its there
> ![Annotation 2019-07-19 134346](https://user-images.githubusercontent.com/5259926/61520198-49347100-aa2b-11e9-85eb-ef71abc115cb.jpg)

### Its hidden, user cant find easily this button, so after save please so popup which shows go to user area button.
